### DISCUSSION TITLE
2023-09-25 정기회의

### DISCUSSION THREAD

**Discussion by scarf005** [jiphyeonjeon-42/backend#781] [General]
안건을 자유롭게 추가해주세요.

**Comment by Jiwon-Woo** [comment]
https://github.com/jiphyeonjeon-42/backend/discussions/774#discussioncomment-7019264

저번에 얘기 나눴던 라우트 정리 부분 바로 작업해도 될까요?

**Reply by scarf005** [reply]
넵! 중첩 반환 형태로 작업해주시면 될 것 같아요!

## 중첩 반환 사용 이유
- 구현이 더 간단함
- 프론트에서 id 기반 요청 시 수백 건의 HTTP 요청을 하게 되어 DB에 부하 발생

**Reply by Jiwon-Woo** [reply]
https://github.com/jiphyeonjeon-42/backend/discussions/774#discussioncomment-7019257

중첩반환은 이 코멘트의 `ok` 부분 참고하겠습니다.

**Reply by scarf005** [reply]
넵 알겠습니다!
